cypress/integration/confirmLeave.cy.js

```diff
@@ -11,30 +12,42 @@ import {
 } from '../elements/confirmLeaveModal'
 import {
     clickDimensionModalUpdateButton,
-    removeAllDataItems,
-    selectDataElements,
+    selectRelativePeriods,
 } from '../elements/dimensionModal'
-import { expectNoDataItemsToBeSelected } from '../elements/dimensionModal/dataDimension'
+import { removeAllPeriodItems } from '../elements/dimensionModal/periodDimension'
 import { openDimension } from '../elements/dimensionsPanel'
-import { createNewAO, openRandomAO } from '../elements/fileMenu'
+import { createNewAO, openAOByName } from '../elements/fileMenu'
+import { selectYoyCategoryOption } from '../elements/layout'
+import { clickMenuBarUpdateButton } from '../elements/menuBar'
 import {
     expectStartScreenToBeVisible,
     goToStartPage,
 } from '../elements/startScreen'
-import { TEST_DATA_ELEMENTS } from '../utils/data'
+import { TEST_AOS } from '../utils/data'
 import { getRandomArrayItem } from '../utils/random'
 
 describe('confirm leave modal', () => {
+    const TEST_AO = getRandomArrayItem(TEST_AOS)
+
     it('navigates to the start page and loads a random saved AO', () => {
         goToStartPage()
-        openRandomAO()
+        openAOByName(TEST_AO.name)
         expectAOTitleToNotBeDirty()
     })
-    it('replaces the data items', () => {
-        openDimension(DIMENSION_ID_DATA)
-        removeAllDataItems().then(() => expectNoDataItemsToBeSelected())
-        selectDataElements([getRandomArrayItem(TEST_DATA_ELEMENTS).name])
-        clickDimensionModalUpdateButton()
+    it(`replaces the selected period`, () => {
+        if (isYearOverYear(TEST_AO.type)) {
+            const TEST_PERIOD = 'Last 2 six-months'
+            selectYoyCategoryOption(TEST_PERIOD)
+            clickMenuBarUpdateButton()
+        } else {
+            const TEST_PERIOD_TYPE = 'Six-months'
```

Comment:
Doesn't seem necessary to create consts for values that are only used once? (same with line 39)

Reply:
Yeah good point, it was a copy/paste from a WIP section that I've now moved to a common file and finished 👍